_docs/services/aws-elasticache.md

```diff
@@ -31,6 +31,36 @@ To create a service instance, run the following command (replace `my-ec-service`
 ```sh
 cf create-service aws-elasticache-redis redis-3node my-ec-service
 ```
+### Setting optional parameters
+
+There are several optional parameters that you can specify when creating a new instance:
+
+Name               | Description                                                    | Default Value          |
+---                | ---                                                            | ---                    |
+`engineVersion`          | Specifies a supported major version in Redis (must be in "")   | 6.2                    |
+
+A couple of notes regarding the optional `version` parameter:
+
+- It only supports major version numbers; if you specify a minor/patch level version, e.g., "6.2.1" , the command will fail.
+- The version number must be provided in double quotes (`"`); this is because the value is treated as a string to account for different versions.
+
+These are the current supported major versions for Redis:
+
+- 5.0.6
```

Comment:
Should this be listed as 5.0 since this is a list of supported major versions?

Reply:
https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/supported-engine-versions.html
its more abstract for versions 6 and 7 but is longer for 5.0.x

Reply:
so version 5.0.6 is different from 5.0.5